frontend/src/pages/About.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {NavLink} from "react-router-dom";
+
+export default function About() {
+    return (
+        <div>
```

Comment:
maybe u can use a nav element instead of a div since u wan to navigate here to the members page